src/stratigraphy/util/depthcolumn.py

```diff
@@ -321,9 +321,16 @@ def is_valid(self, all_words: list[TextLine]) -> bool:
             > self.noise_count_threshold * (len(self.entries) - self.noise_count_offset) ** 2
         ):
             return False
+        # Check if the entries are strictly increasing.
+        if not all(i.value < j.value for i, j in zip(self.entries, self.entries[1:], strict=False)):
+            return False
 
         corr_coef = self.pearson_correlation_coef()
-        return corr_coef and corr_coef > 0.99
+
+        return (
+            corr_coef and corr_coef > np.min([1.0382 - len(self.entries) * 0.01, 0.9985]) and corr_coef > 0.95
```

Comment:
Admittably, this is probably overfitted. Will be interesting to see the real scores ones we have a test set.